src/components/FormStepSummary/ComponentValueDisplay.js

```diff
@@ -214,9 +214,14 @@ const AddressNLDisplay = ({component, value}) => {
     return <EmptyDisplay />;
   }
 
-  return `${value.postcode} ${value.houseNumber}${value.houseLetter || ''} ${
-    value.houseNumberAddition || ''
-  }`;
+  return (
+    <>
+      {value.streetName} {value.houseNumber}
+      {value.houseLetter} {value.houseNumberAddition}
+      <br />
+      {value.postcode} {value.city}
```

Comment:
these should probably only be displayed if `deriveAddress` is enabeld, and if their values are empty maybe don't display them?